tests/checkpoint/test_checkpoint.py

```diff
@@ -80,146 +73,6 @@ def batch_request_as_dict() -> Dict[str, str]:
     }
 
 
-@pytest.mark.filesystem
-def test_basic_checkpoint_config_validation(
```

Comment:
I've deleted various tests within this file that focus on behavior that is specific to the legacy checkpoint - please take a look at the test names and confirm that I'm correct in deleting these!

We'll probably need to do another pass but I'm just cleaning up as I go.